rel/overlay/etc/default.ini

```diff
@@ -91,6 +91,10 @@ delete_dbs = false
 ; Set a default q value for peruser-created databases that is different from
 ; cluster / q
 ;q = 1
+; prefix for user databases. If you change this after user dbs have been
+; created, the existing databases won’t get deleted if the associated user
```

Comment:
Hi @jiangphcn , this line broke the Jenkins build.

Your apostrophe in `won't` is actually a "smartquote", probably from your Mac.

Older Pythons can't parse these, so `dev/run` failed on reading the file to create a new config file:

```
Traceback (most recent call last):
  File "dev/run", line 572, in <module>
    main()
  File "dev/run", line 80, in main
    ctx = setup()
  File "dev/run", line 94, in setup
    setup_configs(ctx)
  File "dev/run", line 65, in wrapper
    res = func(*args, **kwargs)
  File "dev/run", line 200, in setup_configs
    write_config(ctx, node, env)
  File "dev/run", line 230, in write_config
    content = handle.read()
  File "/usr/lib64/python3.4/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 3976: ordinal not in range(128)
```

Be sure to check for this in the future :)

Reply:
@wohali thanks for finding this. I found that you already fixed it. Thanks. Yes, will remember this and make it not happen in the future :-)